SU-HSDO/suhumsci
# fix(STN-492): secondary nav header spacing and border-top

# READY FOR REVIEW

## Summary
Fixes an issue with the secondary nav on traditional theme where spacing looks wrong because it was, and it also didn't have a border-top.

⚠️ This PR cannot be merged until the sparkbox-sprint-23 branch has been created.

- [x] ran backstop js

## Steps to Test
1. `npm run test` passes
2. Check that nothing has changed for Colorful theme on the secondary nav.
3. Check that the traditional theme secondary nav matches the mockups now:  https://www.figma.com/file/bG42fKKioEcnJ0ORKkLL5E/%E2%9A%99%EF%B8%8F-Traditional-Theme---Working?node-id=5686%3A2

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
